x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_custom_source/configure_custom.tsx

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ChangeEvent, FormEvent } from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiButton,
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiFormRow,
+  EuiLink,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { docLinks } from '../../../../../../shared/doc_links';
+
+import connectionIllustration from '../../../../../assets/connection_illustration.svg';
+import { SOURCE_NAME_LABEL } from '../../../constants';
+
+import { AddSourceHeader } from '../add_source_header';
+import { CONFIG_CUSTOM_BUTTON, CONFIG_CUSTOM_LINK_TEXT, CONFIG_INTRO_ALT_TEXT } from '../constants';
+
+import { AddCustomSourceLogic } from './add_custom_source_logic';
+
+export const ConfigureCustom: React.FC = () => {
+  const { setCustomSourceNameValue, createContentSource } = useActions(AddCustomSourceLogic);
+  const { customSourceNameValue, buttonLoading, sourceData } = useValues(AddCustomSourceLogic);
+
+  const handleFormSubmit = (e: FormEvent) => {
+    e.preventDefault();
+    createContentSource();
+  };
+
+  const handleNameChange = (e: ChangeEvent<HTMLInputElement>) =>
+    setCustomSourceNameValue(e.target.value);
+
+  const {
+    serviceType,
+    configuration: { documentationUrl, githubRepository },
+    name,
+    categories = [],
+  } = sourceData;
+
+  return (
+    <>
+      <AddSourceHeader name={name} serviceType={serviceType} categories={categories} />
+      <EuiSpacer />
+      <EuiFlexGroup
+        justifyContent="flexStart"
+        alignItems="stretch"
+        direction="row"
+        gutterSize="xl"
+        responsive={false}
+      >
+        <EuiFlexItem grow={false}>
+          <div className="adding-a-source__intro-image">
+            <img src={connectionIllustration} alt={CONFIG_INTRO_ALT_TEXT} />
+          </div>
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiSpacer size="xl" />
+          <EuiTitle size="l">
+            <h2>
+              <FormattedMessage
+                id="xpack.enterpriseSearch.workplaceSearch.contentSource.configCustom.title"
+                defaultMessage="How to add {name}"
+                values={{ name }}
+              />
+            </h2>
+          </EuiTitle>
+          <EuiSpacer size="m" />
+          <form onSubmit={handleFormSubmit}>
+            <EuiForm>
```

Comment:
**No need to fix it here – just future reference.** I believe you can pass `component='form'` as a prop on the `EuiForm` component in order to avoid having to wrap the `EuiForm` in a separate `form` tag.